### DISCUSSION TITLE
Some links is 404 in README

### DISCUSSION THREAD

**Discussion by yansheng836** [apache/cloudberry#704] [Q&A]
Part of the link on the home page is 404, I guess it is caused by the project's migration to Apache, I suggest replacing it as a whole.

like the roadmap link is 404:

https://github.com/orgs/cloudberrydb/discussions/369

**Comment by tuhaihe** [comment]
Hi @yansheng836 the latest link should be https://github.com/apache/cloudberry/discussions/369 now.

**Reply by tuhaihe** [reply]
We're working on a new site - https://github.com/apache/cloudberry-site/pull/170. Once this PR is merged, the new website will go live.

**Comment by yansheng836** [comment]
Thanks.
